config/config.go

```diff
@@ -6,19 +6,26 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
-// TODO: remove hello call config options
+// KafkaTLSProtocolFlag informs service to use TLS protocol for kafka
+const KafkaTLSProtocolFlag = "TLS"
```

Comment:
Im not sure why this is needed?

Default is empty string `""` and then if we want TLS then we set the `KAFKA_SEC_PROTO` to be `"TLS"`

Reply:
Okay further looking into this - this isn't a flag, it just describes a kafka protocol that is used to compare against the actual flag `KafkaSecProtocol`.

The description doesn't help decipher what this actually is.

```suggestion
// KafkaTLSProtocol is a constant describing the TLS protocol used for kafka
const KafkaTLSProtocol = "TLS"
```